### PR TITLE
chore: update unirep to 2.0.0 beta 6

### PR DESCRIPTION
**Pull request by ryanycw**
## Summary

Update Unirep to 2.0.0 beta 6

## Checklist

-   [x] All new and existing tests pass
-   [ ] I have updated the documentation (if applicable)
-   [x] My changes do not introduce new security vulnerabilities
-   [x] Run `yarn lint:fix`

### CODE REVIEW

File: packages/relay/package.json [hunk 1]
```diff
@@ -17,8 +17,7 @@
         "@helia/json": "^1.0.0",
         "@types/crypto-js": "^4.1.1",
         "@unirep-app/contracts": "1.0.0",
-        "@unirep/core": "2.0.0-beta-4",
-        "anondb": "^0.0.13",
```
Comment (chentihe) [inline packages/relay/package.json:21]: anondb should keep in relay packages

File: packages/circuits/package.json [hunk 2]
```diff
@@ -4,34 +4,30 @@
     "main": "dist/src/index.js",
     "types": "dist/src/index.d.ts",
     "description": "ZK proofs used for the unirep attesters",
-    "author": "Unirep social TW",
-    "homepage": "https://github.com/social-tw/social-tw-website",
+    "author": "Unirep Team <team@unirep.io>",
```
Comment (chentihe) [inline packages/circuits/package.json:7]: should modify to previous description

File: packages/circuits/package.json [hunk 3]
```diff
@@ -4,34 +4,30 @@
...
+    "homepage": "https://github.com/Unirep/create-unirep-app#readme",
     "license": "MIT",
     "repository": {
         "type": "git",
-        "url": "https://github.com/social-tw/social-tw-website.git"
+        "url": "git+https://github.com/Unirep/create-unirep-app.git"
```
Comment (chentihe) [inline packages/circuits/package.json:12]: should modify to previous description

File: packages/circuits/package.json [hunk 4]
```diff
@@ -4,34 +4,30 @@
...
     "scripts": {
         "build": "tsc && yarn copyCircom",
-        "buildOverwrite": "find ./zksnarkBuild -type f -not -name '*.ptau' -delete && yarn buildsnark",
         "buildsnark": "ts-node ./scripts/buildSnarks.ts && tsc && yarn copyCircom",
         "test": "mocha -r ts-node/register test/*.test.ts --exit",
         "copyCircom": "sh scripts/copyCircom.sh"
     },
     "bugs": {
-        "url": "https://github.com/social-tw/social-tw-website/issues"
+        "url": "https://github.com/Unirep/create-unirep-app/issues"
```
Comment (chentihe) [inline packages/circuits/package.json:21]: should modify to previous description

File: packages/relay/package.json (hunk 1 above)
Comment (ryanycw) [reply packages/relay/package.json:21]: Thank you for the feedback, it's been moved to "@unirep/core": "2.0.0-beta-6"

File: packages/circuits/package.json (hunk 4 above)
Comment (ryanycw) [reply packages/circuits/package.json]: Thank you! I believe it's all fixed, could you take a look again

**Review by chentihe — APPROVED** [review]
LGTM

**Pull request merged**